feature-utils/poly-look/src/react-components/filterChips.jsx

```diff
@@ -61,13 +61,26 @@ const FilterChips = ({
         )
   );
 
-  const [initialDefaultChips, setInitialDefaultChips] =
-    useState(defaultActiveChips);
+  const isFirstRender = useRef(true);
+  const initialActiveChips = useRef(defaultActiveChips);
 
-  if (defaultActiveChips !== initialDefaultChips) {
-    setActiveChips(defaultActiveChips);
-    setInitialDefaultChips(defaultActiveChips);
-  }
+  useEffect(() => {
+    if (isFirstRender.current) {
+      isFirstRender.current = false;
+      return;
+    }
```

Comment:
AFAIK, with this, the functionality of being able to change `defaultActiveChips` later is lost. Doesn't it work the same without this? The comparison just checks if `defaultActiveChips` changed.

Reply:
This just makes sure we don't do the check on first render. But I guess it works without as well.